types.go

```diff
@@ -738,6 +738,20 @@ func Visibility(v VisibilityValue) *VisibilityValue {
 	return p
 }
 
+// VisibilityLevelValue represents a visibility level within GitLab in int type.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/
+type VisibilityLevelValue int
+
+// List of available visibility levels.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/
+const (
+	PrivateVisibilityLevel  VisibilityLevelValue = 0
+	InternalVisibilityLevel VisibilityLevelValue = 10
+	PublicVisibilityLevel   VisibilityLevelValue = 20
+)
+
```

Comment:
```go
// VisibilityLevel is a helper routine that allocates a new
// VisibilityLevelValue to store v and returns a pointer to it.
func VisibilityLevel(v VisibilityLevelValue) *VisibilityLevelValue {
	p := new(VisibilityLevelValue)
	*p = v
	return p
}
```

Can you add something like this ☝🏻 I try to add such a helper function for all custom types to make it easier for people to get a pointer to such a value (which is needed in option structs).